themes/default/content/blog/meals-for-ukraine/index.md

```diff
@@ -0,0 +1,58 @@
+
+Of course, there are many more reasons to visit our booths. You'll find lots of great cloud native resources including demos, tutorials, ebooks, and more. Here are some of the highlights you can expect at each booth.
+
+### Buoyant — the creator of Linkerd
+
+[Buoyant](https://buoyant.io/) is announcing something really exciting: the introduction of *fully managed Linkerd*. You can be one of the first to check out the demo, and you'll also find a ton of service mesh content including a service mesh 101 in English and Spanish , our engineer's guide to mTLS, and details on Buoyant's Service Mesh Academy. For this and more, visit [the Buoyant booth](https://kubecon-cloudnativecon-europe.com/virtual-exhibitor/?v4b4342b0f72f3260e37d74de68eab433fee0c641d933e76d52be7eb34b211371656f07b5a54b2e522db3ac7b27c7d555=908815D44FE61A2E80915ACEC6E64FC9A625809715300E330E9CA19100987FDC5540D5D12FFF991541BCDBA641CF8FDC)!
+
+### Civo — Fast Simple, Managed Kubernetes
+
+Stop by the [Civo](https://www.civo.com/) booth for live sessions on how to get the most out of your Kubernetes environment - including demos, Civo marketplace walkthroughs and an inside look at how they live-provision a private region. You'll find lots of resources and friendly team members to answer any questions on Civo’s easy-to-use managed Kubernetes service. It is fully CNCF certified and 100% Kubernetes upstream compatible. [Check out the Civo booth today](http://civo.io/kubecon)!
+
+## Cockroach Labs — Build What you Dream
+
+[CockroachDB](https://www.cockroachlabs.com/)'s presence this year at KubeCon (both physical and virtual) is primarily a charitable one (if you attend in person, visit booth P16, where for each badge scanned they'll donate $3 to either Black Girls Code, Women Who Code, the Cancer Research Institute, or UNICEF for Ukrainian relief). Visit their booth ([or book a meeting here](https://www.cockroachlabs.com/guides/kubecon-emea-2022/)) to see demos and architecture overviews that relate what they’ve learned from building a multi-region, managed distributed SQL database offering on Kubernetes. [Check it out now](http://cockroachlabs.com/kubecon).
+
+### Fairwinds — Kubernetes Governance and Security
+
+[Fairwinds](http://www.fairwinds.com), a leader in Kubernetes governance providing a unified view between dev, sec, and ops, offers lots of great resources at their virtual booth this KubeCon. You can get a quick tour of Fairwinds Insights, learn about Kubernetes cost savings, how to make Kubernetes service ownership possible, typical Kubernetes security mistakes and much more. Come chat with a friendly Fairwinds engineer to talk about the many great open source solutions including Polaris, Goldilocks, Nova, and Pluto.  [Visit the Fairwinds booth today](https://fairwinds.com/kubecon)!
+
+### Pulumi — Universal Infrastructure as Code
```

Comment:
Given this list isn't in alphabetical order anyway, why not move Pulumi to the top of the list so people see it easier

Reply:
Changed the order in [`91642ab` (#1501)](https://github.com/pulumi/pulumi-hugo/pull/1501/commits/91642ab3f66c17c3b3162afb619ecdec9db8520b)